student/tests/management/commands/test_create_slack_section_channels.py

```diff
@@ -0,0 +1,61 @@
+import datetime
+from django.conf import settings
+import pytest
+
+from staff.models.batch import Batch
+from staff.models.course import Course
+from staff.models.section import Section
+from student.library.slack import Slack
+from student.management.commands.create_slack_section_channels import Command
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.fixture()
+def batch():
```

Comment:
Should this mock function be named more explicitly, e.g. `create_test_batch`? I don't have strong opinions

Reply:
i think batch is ok since it yields a batch at the end and i want to use the object in this way `batch.number`

Reply:
SGTM